mobx_codegen/test/data/valid_annotated_store_referencing_store_input.dart

```diff
@@ -0,0 +1,50 @@
+library generator_sample;
+
+import 'dart:ui' as ui;
+
+import 'package:mobx/mobx.dart';
+
+part 'generator_sample.g.dart';
```

Comment:
I think we can remove the part file declarations from all input files. Tests should still work I think

Reply:
As it turns out, we cannot. In their absence, generator is reported to be an empty string by the tests.